platform/ios/PolyPodApp/Utils/Array+Utils.swift

```diff
@@ -0,0 +1,7 @@
+extension Array {
```

Comment:
Nit: So far we've had all the extensions to standard types in a dedicated directory, wouldn't it be cleaner if we moved this one there too?